pkgs/applications/science/math/symmetrica/default.nix

```diff
@@ -34,13 +29,34 @@ stdenv.mkDerivation rec {
         url = "https://git.sagemath.org/sage.git/plain/build/pkgs/symmetrica/patches/int32.patch?id=07d6c37d18811e2b377a9689790a7c5e24da16ba";
         sha256 = "0p33c85ck4kd453z687ni4bdcqr1pqx2756j7aq11bf63vjz4cyz";
       })
+
+      (fetchpatch {
+        url = "https://git.sagemath.org/sage.git/plain/build/pkgs/symmetrica/patches/return_values.patch";
```

Comment:
Can you use a commit instead of master to load this patch to avoid breakages if the source gets an update?